src/lava/proc/lif/process.py

```diff
@@ -7,7 +7,29 @@
 from lava.magma.core.process.ports.ports import InPort, OutPort
 
 
-class LIF(AbstractProcess):
+class AbstractLIF(AbstractProcess):
+    """Abstract class for variables common to all neurons with leaky
+    integrator dynamics."""
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        shape = kwargs.get("shape", (1,))
+        du = kwargs.pop("du", 0)
+        dv = kwargs.pop("dv", 0)
+        bias = kwargs.pop("bias", 0)
+        bias_exp = kwargs.pop("bias_exp", 0)
+
+        self.shape = shape
+        self.a_in = InPort(shape=shape)
+        self.s_out = OutPort(shape=shape)
+        self.u = Var(shape=shape, init=0)
+        self.v = Var(shape=shape, init=0)
+        self.du = Var(shape=(1,), init=du)
+        self.dv = Var(shape=(1,), init=dv)
+        self.bias = Var(shape=shape, init=bias)
```

Comment:
Long term, if we are to keep with the abstract inheritance structure you've started here for neuron procs, I think we should make bias and bias_exp part of a fixed-point precision mix-in for child LIF class rather than default vars in the AbstractLIF class, since bias and bias_exp only pertain to fixed-point implementations of LIF. 

But, this implementation is probably ok while we are still debating the best design patterns for neuron Procs.